source/includes/container/steps-configure-keycloak-identity-management.rst

```diff
@@ -0,0 +1,324 @@
+1) Create the Podman Pod
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create a Podman Pod to deploy the Keycloak and MinIO containers in a Pod with shared networking.
+This ensures both containers can communicate normally.
+
+.. code-block:: shell
+   :class: copyable
+
+   podman pod create \ 
+        -p 9000:9000 -p 9090:9090 -p 8080:8080 \
+        -v ~/minio-keycloak/minio:/mnt/minio \
+        -n minio-keycloak
+
+Replace ``~/minio-keycloak/minio`` with a path to an empty folder in which the MinIO container stores data.
+
+You can alternatively deploy the Containers as Root to allow access to the host network for the purpose of inter-container networking.
+
+Deploying via Docker Compose is out of scope for this tutorial.
+
+2) Start the Keycloak Container
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow the instructions for running `Keycloak in a container <https://www.keycloak.org/server/containers>`__.
+The `Try Keycloak  in development mode <https://www.keycloak.org/server/containers#_trying_keycloak_in_development_mode>`__ steps are sufficient for this procedure.
+
+.. code-block:: shell
+   :class: copyable
+
+   podman run -dt \
+          --name keycloak \
+          --pod minio-keycloak \
+          -e KEYCLOAK_ADMIN=keycloakadmin \
+          -e KEYCLOAK_ADMIN_PASSWORD=keycloakadmin123 \
+          quay.io/keycloak/keycloak:latest start-dev
+
+You can access the Keycloak container over the ``http://localhost:8080`` in your browser.
+
+3) Configure or Create a Client for Accessing Keycloak
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
The `guilabel` and monotype doesn't work for these lists. It's really hard to parse.
It's just a sea of gray boxes.

Field lists or tables instead?

![image](https://user-images.githubusercontent.com/53910321/222578196-9db51c5c-ebcb-4747-9daa-e8dae0f9daf9.png)